spec/truffle/interop/key_info_spec.rb

```diff
@@ -10,6 +10,90 @@
 require_relative 'fixtures/classes'
 
 describe "Truffle::Interop.key_info" do
+  describe "for an Array" do
+    
+    before :each do
+      @array = [1, 2, 3]
+    end
+    
+    it "returns :readable for indicies in bounds" do
```

Comment:
"indicies" -> "indices"? (Throughout this file if you agree).

Reply:
Fixed.